cfg/validators-example.txt

```diff
@@ -39,6 +43,11 @@
 #   publisher key.
 #   Validator list keys should be hex-encoded.
 #
+#   The latest list of recommended validator sites and keys can be
```

Comment:
Tweak: `The latest list of recommended validator keys can be`

Reply:
Same